Periodic_3_triangulation_3/demo/Periodic_3_triangulation_3/Scene.cpp

```diff
@@ -1413,7 +1413,7 @@ void Scene::gl_draw_location() {
                 Point p = tet_to_draw.vertex((i+1)&3);
                 Point q = tet_to_draw.vertex((i+2)&3);
                 Point r = tet_to_draw.vertex((i+3)&3);
-                Vector c= (Vector(Point(),p)+Vector(Point(),q)+Vector(Point(),r))/3.;
+                Vector c= (Vector(Point(0,0,0),p)+Vector(Point(0,0,0),q)+Vector(Point(0,0,0),r))/3.;
                 Point cp = Point(c.x(),c.y(),c.z());
```

Comment:
I think a better code would be:
```c++
Point cp = CGAL::centroid(p, q, r);
```

https://doc.cgal.org/latest/Kernel_23/group__centroid__grp.html

https://doc.cgal.org/latest/Kernel_23/classKernel_1_1ConstructCentroid__3.html

Reply:
Just did it. I was blind by only thinking about fixing the "uninitialized" warning.